backend/main.py

```diff
@@ -21,62 +17,46 @@ class StandardResponse(BaseModel):
     data: Any = None
 
 
-if os.path.exists(".env"):
-    load_dotenv(".env")
-
-redis_client = redis.Redis(host="redis", port=6379, db=0)
-
-spotihue = spotihue.SpotiHue(
-    os.environ.get("SPOTIFY_SCOPE"),
-    os.environ.get("SPOTIFY_CLIENT_ID"),
-    os.environ.get("SPOTIFY_CLIENT_SECRET"),
-    os.environ.get("SPOTIFY_REDIRECT_URI"),
-    os.environ.get("HUE_BRIDGE_IP_ADDRESS"),
-    redis_client
-)
-
-celery_app = celery.Celery("celery_app", broker="redis://redis:6379")
 fast_app = FastAPI()
 
 
-@celery_app.task
-def run_spotihue(lights: List[str]) -> None:
-    logger.info("Running spotihue")
+@fast_app.get("/hue-ready")
```

Comment:
Would it make sense to combine hue ready and setup into one endpoint? Checks if hue_ready() then if not goes through setup? Not sure on what is best here...

Reply:
Ah, yeah, that'd probably make more sense. It does look like the frontend will need to know both statuses at the same time (and not in separate pages), looking at the set-up-1 Figma view.
I can change these two to just one "ready" endpoint, with a
`{"spotify_ready": <true/false>, "hue_ready": <true/false>} `sort of response data!